Website/src/rendering/src/components/Speakers/FeaturedSpeakers.tsx

```diff
@@ -24,26 +35,31 @@ const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => {
     props.fields.data.item.children.results
       .filter((item) => item.featured.value)
       .sort()
-      .slice(0, parseInt(props.params.NumberOfSpeakers))
+      .slice(0, getSpeakerNumberToShow(props))
       .map((speaker, index) => (
         <Link key={index} href={`/speakers/${speaker.name.value}`} passHref>
-          <a className="grid-item">
-            <Image
-              field={speaker.picture.jsonValue}
-              alt={speaker.name}
-              width={265}
-              height={265}
-              loading="lazy"
-            />
-            <div className="item-details">
-              <Text tag="p" field={speaker.name} />
+          <a>
+            <div className="grid-item">
+              <div className="item-image">
+                <Image
+                  field={speaker.picture.jsonValue}
+                  alt={speaker.name.value}
+                  width={265}
+                  height={265}
+                  loading="lazy"
+                />
+              </div>
+              <div className="item-details">
+                <Text tag="p" className="item-title" field={speaker.name} />
```

Comment:
Featured speakers name does not have the same top/bottom margins as regular speakers.

![image](https://user-images.githubusercontent.com/2000200/151561923-dab1f2bf-be20-461f-840d-297059cf3ad8.png)